docs/model_overview.md

```diff
@@ -13,6 +13,7 @@ The table below further shows which model architectures support which adaptation
 
 | Model                                   | (Bottleneck)<br> Adapters | Prefix<br> Tuning | LoRA | Compacter | Adapter<br> Fusion | Invertible<br> Adapters | Parallel<br> block | Prompt<br> Tuning | ReFT |
 | --------------------------------------- | -| - | - | - | - | - | - |- | - |
+| [Custom models](plugin_interface.html)    | ✅(°) |  | ✅ | ✅ | ✅ | ✅ |  | ✅ | ✅ |
```

Comment:
Once we update this PR to include some model interfaces out of the box (like Gemma), we should list them here too and mention them in docs/plugin_interface.md

I believe this is a blocker for merging - we need to integrate these model interfaces before we merge.

Reply:
I think it's perfectly fine to provide the built-in interfaces in a separate PR. This way we also don't make a single huge change as this PR is already quite big.